### ISSUE TITLE
Sponsorship tooling

### ISSUE THREAD

**Issue by jmduke** [buttondown/roadmap#1502]
## TL;DR

![image](https://github.com/buttondown-email/roadmap/assets/461046/89aff074-6bac-45b0-ba1f-dc2d8054acec)

## Work to be done

- [x] Allow opt-in for sponsorship view
- [x] Allow authors to create SKUs
- [x] Allow authors to mark specific slots as pending/sold
- [x] Add 'success' page for purchased slots
- [x] Email authors when a slot has been sold
- [x] Surface invoices in detail page for sold slots
- [ ] Allow configuring which statistics are shown
- [ ] Allow RSS subscriber count as a statistic

## Out of scope for now, but interesting ideas

- [ ] Handling demand-side (surfacing empty slots to advertisers and holding inventory)
- [ ] "Notify me when a sponsorship is available for this newsletter"

**Comment by jmduke** [comment]
@stefanjudis This is a great angle that I don't think I've thought about enough! I'd definitely want to make sure it's well-supported.

Right now, the flow for you would be something like: you link from `webweekly.email` to `https://buttondown.email/stefanjudis/sponsorships`, which would then have a slot open for each week.  Buttondown then manages the links for those invoices, confirmation, etc.

Is it a hard no for you to have folks go to `https://buttondown.email/stefanjudis/sponsorships` to handle that part, since you'd want styling and all that? I don't have any problem with that from a _branding_ standpoint, but it makes me wonder what the easiest way for you to integrate would be — maybe just pulling the open dates + URLs from `api.buttondown.email/v1/advertising-slots`?

**Comment by stefanjudis** [comment]
@jmduke Thanks for considering.

> Is it a hard no for you to have folks go to https://buttondown.email/stefanjudis/sponsorships to handle that part, since you'd want styling and all that?

It isn't. Your thinking is spot on.

My preferred solution would be:

- have `webweekly.email/sponsorships` that displays open slots, price, conditions and stats via API (all read only)
- link to `https://buttondown.email/stefanjudis/sponsorships` to receive and handle a booking

If `https://buttondown.email/stefanjudis/sponsorships` gives the possibility for a logo / hero on top, that'd work for me.

**Comment by jmduke** [comment]
@stefanjudis You got it. All should be doable.

**Comment by jmduke** [comment]
Some good examples via @matthiasott of sponsorship pages:

- https://css-weekly.com/advertise/
- https://www.densediscovery.com/advertise/
